### PR TITLE
improve ux with hints for serve all requests for the `/static/` and `/media/` paths

### PR DESCRIPTION
**Pull request by turbotimon**
According to #56

### CODE REVIEW

File: conf/pretalx.cfg [hunk 1]
```diff
@@ -2,7 +2,9 @@
 # use PRETALX_FILESYSTEM_* env vars
 
 [site]
-; never run debug in production
+; never run debug in production. Make sure you serve all requests for the
+; `/static/` and `/media/` paths when debug is False.
+; See [installation](https://docs.pretalx.org/administrator/installation/#step-7-ssl) for more information
```
Comment (rixx) [inline conf/pretalx.cfg:7]: Not sure it's particularly useful to have a markdown-formatted link in a file that will never be parsed as markdown, and will make the link harder to select and open in most file editors

File: docker-compose.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ services:
       - redis
       - db
     environment:
+      # Hint: Make sure you serve all requests for the `/static/` and `/media/` paths when debug is False. See [installation](https://docs.pretalx.org/administrator/installation/#step-7-ssl) for more information
```
Comment (rixx) [inline docker-compose.yml:11]: I'm not convinced that adding this informaitn here is valuable – you might as well add information on every other environment variable and/or setting and/or component.

**Review by rixx — APPROVED** [review]
Thanks! Not convinced that this will help too much – feels like a very narrow reaction to one problem, so but as I keep saying: this repo is community-maintained and I'm happy to bow to community preferences.

File: conf/pretalx.cfg (hunk 1 above)
Comment (turbotimon) [reply conf/pretalx.cfg:7]: Yes, you're absolutely right about the markdown... Should i remove it?

**Pull request merged**